assets/src/state.ts

```diff
@@ -532,9 +561,21 @@ const routeTabsReducer = (
           newRouteTab(state.length),
         ],
       ]
+    case "CREATE_PRESET":
+      return [state, [...state, newPreset(action.payload.sourceTab)]]
```

Comment:
**issue:** as it stands, this logic means the tab you have open will stay "untitled" rather than becoming an instantiation of the new preset. This seems to contradict the clickable Figma prototype, which shows the tab changing from "Untitled" to "Cabot-1" once the preset is saved.

Reply:
Note that I think this is easy to fix - one option is to go with the suggestion in my comment below, but it's probably also possible to just carry the `presetName` from the new preset back over to the current tab.